charts/ceph-csi-cephfs/Chart.yaml

```diff
@@ -1,15 +1,15 @@
 ---
 apiVersion: v1
-appVersion: "1.3.0"
+appVersion: canary
 description: "Container Storage Interface (CSI) driver,
 provisioner, and attacher for Ceph cephfs"
 name: ceph-csi-cephfs
-version: 0.9.0
+version: 1.3.0-canary
```

Comment:
instead of 1.3.0 can this be canary?

Reply:
No, helm enforces semver on this field and a string isn't valid semver

Reply:
So before a minor version release these values need to be updated. Sadly I can't make much more of this